Explorer/Assets/Scripts/SceneRuntime/Factory/SceneRuntimeFactory.cs

```diff
@@ -24,6 +25,26 @@ public enum InstantiationBehavior
         private readonly JsCodeResolver codeContentResolver;
         private readonly Dictionary<string, string> sourceCodeCache;
 
+        private static readonly IReadOnlyCollection<string> JS_MODULE_NAMES;
+        private readonly IJsSceneSourceCode jsSceneSourceCode =
+            Application.isEditor
+                ? new LogJsSceneSourceCode(
+                    new StreamingAssetsJsSceneSourceCode(),
+                    ReportHub.WithReport(ReportCategory.SCENE_FACTORY).Log
+                )
+                : new IJsSceneSourceCode.Null();
+
+        static SceneRuntimeFactory()
+        {
+            JS_MODULE_NAMES = Directory
```

Comment:
Let's move it from the static constructor, it looks dirty, move it to the container.
Moreover, I would not recommend using any `Path/Directory` API as it does not work on some platforms (not very relevant to us right now but still), it's better to use WebRequest for that,

Reply:
does WebRequest support fetching directory content?